src/Http/Controllers/OpenApiJsonController.php

```diff
@@ -8,19 +8,26 @@
 
 class OpenApiJsonController
 {
-    public function __invoke() : JsonResponse
+    public function __invoke(string $path, string $filename) : JsonResponse
     {
-        $json = $this->getJson();
+        $file = collect(config('swagger-ui.files'))->filter(function ($values) use ($filename, $path) {
```

Comment:
can you add tests for this?

Reply:
can you add tests:
- it_returns_not_found_response_if_provided_version_does_not_exist_in_file
- it_returns_not_found_response_if_provided_file_does_not_exist (can you add this one for the swagger index route as well?)